examples/control_flow.rs

```diff
@@ -88,7 +88,7 @@ fn main() {
                     window.request_redraw();
                 }
                 if close_requested {
-                    *control_flow = ControlFlow::Exit;
+                    *control_flow = ControlFlow::Exit(0);
```

Comment:
That indeed does make sense, this adds a direct connection to "success" and `0` instead of letting an user who never worked with exit codes in the dark. I'm not sure if this would be better as an associated constant though, a function like that looks it would actually do something more special imo.

Reply:
Now with `Exit` being an alias for `ExitWithCode(0)`, I don't know how useful this is. Would a `ExitFailure` (can't think of anything more creative) be more confusing than useful for this?

Reply:
Nah, I don't think `ExitFailure` or similar is needed